### DISCUSSION TITLE
Widget NextJS for external websites

### DISCUSSION THREAD

**Discussion by frietkip** [vercel/next.js#30888] [Help]
I need to build an **embeddable widget** (no iframe) for our Next JS application so our clients can run the Next JS application on their websites. I see this request has been asked before via this [thread](https://spectrum.chat/next-js/general/hello-guys-i-just-wondering-about-if-it-is-possibble-to-use-next-js-to-write-standalone-widget-and-then-embed-it-to-third-part-site-as-for-example-facebook-sharing-like-button-thanks-for-any-advices~7ad7d4ca-da31-4d02-9544-69f21684f424) but I was hoping if there is an update on this. I know @fouad was working on this back in 2020. 

Ideally to be used by a third-party like so:
```
<div id="example"></div>
<script src="https://example.com/widgets/example.bundle.js"></script>
<script>
Example.init({
    selector: "#example",
    name: "example"
});
</script>
```

Any help or suggestion is appreciated. Thanks!

**Comment by nithin-murali-arch** [comment]
Did you find a way?
